common/src/main/java/com/distkv/dst/common/entity/sortedList/SortedListEntity.java

```diff
@@ -28,7 +28,27 @@ public void setMember(String info) {
 
   @Override
   public int compareTo(SortedListEntity o) {
-    return o.getScore() - getScore();
+    if (o.getScore() != getScore()) {
+      return o.getScore() - getScore();
+    }
+    return getMember().compareTo(o.getMember());
```

Comment:
why compare members?

Reply:
We have to make a clear **SortedList** order. When members of our list have equal scores, we need to consider how members are sorted. So there must be clear.

Reply:
OK, gotit.